pest/src/error.rs

```diff
@@ -230,11 +237,17 @@ impl<R: RuleType> Error<R> {
         self
     }
 
+    fn start(&self) -> (usize, usize) {
+        match self.line_col {
+            LineColLocation::Pos(line_col) => line_col,
+            LineColLocation::Span(start_line_col, _) => start_line_col
+        }
+    }
+
     fn spacing(&self) -> String {
-        let line = if let Some((line, _)) = self.end {
-            cmp::max(self.start.0, line)
-        } else {
-            self.start.0
+        let line = match self.line_col {
+            LineColLocation::Pos((line, _)) => line,
+            LineColLocation::Span(_, (line, _)) => cmp::max(self.start().0, line)
```

Comment:
Wouldn't this be better written as

```rust
LineColLocation::Span((head, _), (tail, _)) => cmp::max(head, tail)
```

Reply:
Definitely. It slipped through. Feel free to push the fix and merge if you have the time. I'm going to be travelling until Wednesday.

Reply:
Probably call them `start_line`, `end_line` though?